app/lib/fake_data/complaints.rb

```diff
@@ -0,0 +1,72 @@
+require "ffaker"
+
+module FakeData::Complaints
+  def self.complaint_data
+    {
+      id: FFaker::Random.rand(9999),
+      attributes: {
+        issueLastUpdated: FFaker::Time.date,
```

Comment:
[question] in the response, the dates [should be ISO 8601 formatted](https://jsonapi.org/recommendations/#date-and-time-fields). Do we want to ensure that format here? Or rely on the later `to_json` call to do it for us?

Reply:
I may be misunderstanding your question, but [FFAker's Time](https://github.com/ffaker/ffaker/blob/main/REFERENCE.md#ffakertime) `date` returns `YYYY-MM-DD`, which is ISO 8601.  I do like the idea of specifying in the Open API description for each date that we would like `YYYY-MM-DD` returned, though, to ensure that's what we get back from HSES.

Reply:
Does FFaker return a string that's `YYYY-MM-DD` or a ruby `Date` object? I assumed the latter.

Also, if the time is available this would likely be better as a datetime (more data is better than less) in which case 8601 is: `YYYY-MM-DDTHH:MM:SSZ` (assuming that the time is in UTC timezone, and also I forget how many subseconds are returned)

Reply:
Oh nooo you are correct, I was fooled by their documentation just listing sample dates as though they were strings.  I will mess with this, thanks for catching it.